pkg/registry/core/shoot/strategy.go

```diff
@@ -80,28 +75,6 @@ func (shootStrategy) PrepareForUpdate(ctx context.Context, obj, old runtime.Obje
 	if mustIncreaseGeneration(oldShoot, newShoot) {
 		newShoot.Generation = oldShoot.Generation + 1
 	}
-
-	if utilfeature.DefaultFeatureGate.Enabled(features.ShootMaxTokenExpirationOverwrite) {
-		// Note that this must be executed after `mustIncreaseGeneration` is called, otherwise we might trigger a
-		// reconciliation outside of the maintenance time window because the spec might change in below call.
-		overwriteMaxTokenExpiration(newShoot)
-	}
-}
-
-func overwriteMaxTokenExpiration(shoot *core.Shoot) {
```

Comment:
Why this function is removed? Is the `overwriteMaxTokenExpiration ()` function no longer required ?

Reply:
Yes, it was only required temporarily for overwriting this field for existing shoot clusters.